android/src/main/java/com/thebluealliance/androidclient/viewmodels/ViewModelRenderer.java

```diff
@@ -0,0 +1,11 @@
+package com.thebluealliance.androidclient.viewmodels;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.annotation.WorkerThread;
+
+public interface ViewModelRenderer<VIEWMODEL, ARGS> {
+
+    @WorkerThread
+    @Nullable VIEWMODEL renderToViewModel(Context context, @Nullable ARGS args);
```

Comment:
Ah, I see what you mean. I actually briefly had ViewModels extending from a base class, but then took it out once I realized said class would have absolutely no purpose, short of making it nice to read the code, like you said. I guess we may as well; you have a point about having a common place for future code, even if I can't envision what that would be at this point.